graph/vertex/AttributeVertex.java

```diff
@@ -47,6 +46,12 @@
      */
     VALUE value();
 
+    AttributeVertex.Write<VALUE> writable(); // TODO do we want both of these?
+
+    AttributeVertex.Write<VALUE> asWrite();
```

Comment:
feels a bit odd to have both of these - but one is a cast and one is a conversion